src/controllers/user-controller.js

```diff
@@ -8,7 +8,7 @@ exports.createUser = async (req, res) => {
         const result = await userService.createUser(userData);
         res.status(201).json({ message: 'User berhasil dibuat', userId: result.id });
     } catch (err) {
-        res.status(400).json({ message: err.message });
+        res.status(500).json({ message: err.message });
```

Comment:
ini ga boleh begini.. harusnya 300..

```suggestion
        res.status(502).json({ message: err.message });
```